remix.init/index.js

```diff
@@ -2,26 +2,87 @@ const { execSync } = require("child_process");
 const crypto = require("crypto");
 const fs = require("fs/promises");
 const path = require("path");
-const inquirer = require("inquirer");
 
 const toml = require("@iarna/toml");
-const sort = require("sort-package-json");
+const PackageJson = require("@npmcli/package-json");
+const inquirer = require("inquirer");
+
+const cleanupCypressFiles = (filesEntries) =>
+  filesEntries.flatMap(([filePath, content]) => {
+    const newContent = content
+      .replace(
+        "npx ts-node --require tsconfig-paths/register ./cypress/support/create-user.ts",
+        "node --require tsconfig-paths/register ./cypress/support/create-user.js"
```

Comment:
`tsconfig-paths` with `jsconfig.json` support isn't supported yet, so we need to figure out a way to register `~` paths in the meantime

Reply:
`jsconfig.json` support in `tsconfig-paths` is now released

https://github.com/dividab/tsconfig-paths/blob/master/CHANGELOG.md#400---2022-05-02